src/components/implementation/no_interface/scheddev/fprr/sl_timer_mod_periodic.c

```diff
@@ -0,0 +1,78 @@
+#include <sl.h>
+#include <sl_consts.h>
+#include <sl_plugins.h>
+#include <heap.h>
+
+#define SL_TIMEOUT_MOD_MAX_THDS MAX_NUM_THREADS
+
+struct wakeup_heap {
+	struct heap  h;
+	void        *data[SL_TIMEOUT_MOD_MAX_THDS];
+	char         p; /* pad. TODO: use alignment */
+} wakeup_heap;
+
+static struct heap *wkh = (struct heap *)&wakeup_heap;
```

Comment:
yes, `&wakeup_heap.h`...sorry I got that wrong.

If you want shorthand, use a macro like `SL_TIMER_HEAP()`.  Using another variable just uses memory and creates another structure that might get corrupted.